### PR TITLE
changed repo location

### PR DESCRIPTION
**Pull request by rfarouni**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by rfarouni** [comment]
@bioconda/all Does anyone know why many of the Travis builds  for recent pull requests , including mine, fail?

**Comment by bgruening** [comment]
`six` is missing somewhere. I'm looking at it in the next hours.

**Comment by bgruening** [comment]
Ok, I think I have fixed it. Give it a try.

File: recipes/haystack_bio/meta.yaml [hunk 1]
```diff
@@ -16,12 +16,11 @@ build:
     - "haystack_tf_activity_plane = haystack.generate_tf_activity_plane:main"
     - "haystack_download_genome = haystack.download_genome:main"
     - "haystack_run_test = haystack.haystack_common:run_testdata"
-  number: 1
+  number: 3
```
Comment (bgruening) [inline recipes/haystack_bio/meta.yaml]: this should be 2 isn't it?

Comment (rfarouni) [reply recipes/haystack_bio/meta.yaml]: @bgruening I cancelled the second build that is why. Thanks for fixing it!

**Pull request merged**